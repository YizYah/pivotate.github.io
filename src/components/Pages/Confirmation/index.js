import React from 'react';

const Confirmation = () => {
  return (
    <div className="text-center container full-width">
      <div
        style={{
          margin: 'auto',
          marginTop: '100px',
          maxWidth: '769px',
          backgroundColor: '#ffffff',
          borderRadius: '10px',
          marginBottom: '100px',
        }}
      >
        <img
          style={{ width: '100%' }}
          src="https://pivotatestaticassets.com/images/Banner.jpg"
        />
        <p style={{ textAlign: 'center' }}>
          <br />
          <br />
          <h3 className="c0">
            Your Meeting is Confirmed! Weâ€™ll be talking soon.
          </h3>
          <h4>Check your email for details.</h4>
        </p>
      </div>
      <br />
    </div>
  );
};

export default Confirmation;
