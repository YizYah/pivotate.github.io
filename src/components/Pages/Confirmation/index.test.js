import React from 'react';
import renderer from 'react-test-renderer';
import Confirmation from '../Confirmation';

test('Snapshopt Test', () => {
  const component = renderer.create(<Confirmation />);

  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
