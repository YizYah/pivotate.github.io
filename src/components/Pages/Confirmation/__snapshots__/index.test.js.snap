// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshopt Test 1`] = `
<div
  className="text-center container full-width"
>
  <div
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderRadius": "10px",
        "margin": "auto",
        "marginBottom": "100px",
        "marginTop": "100px",
        "maxWidth": "769px",
      }
    }
  >
    <img
      alt="Banner"
      src="https://pivotatestaticassets.com/images/Banner.jpg"
      style={
        Object {
          "width": "100%",
        }
      }
    />
    <p
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <br />
      <br />
      <h3
        className="c0"
      >
        Your Meeting is Confirmed! Weâ€™ll be talking soon.
      </h3>
      <h4>
        Check your email for details.
      </h4>
    </p>
  </div>
  <br />
</div>
`;
